### DISCUSSION TITLE
About CANVA

### DISCUSSION THREAD

**Discussion by bala04321** [community/community#31938] [GitHub Education]
Hloo;
 I need the pro canva with github but the option was not showing  in the github  student package. It is very important to me and I want to do editing with use of pro cava. Can u pls help me in this.

**Comment by melvincwng** [comment]
Hey @bala04321, unfortunately, Canva Pro is no longer one of the benefits in the GitHub Student Developer Pack.

Hope this clarifies and answers your question 😊

**Comment by bala04321** [comment]
Then what I have use for the editing the photos,pdf can u pls suggest it

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
